docs/users/configuration.md

```diff
@@ -0,0 +1,236 @@
+#Configuring Velocity
+Velocity has been designed to be simple and unambiguous to configure.
+
+##The configuration file
+
+Velocity is largely configured from the `velocity.toml` file. This file is created in the directory where you started
+ the proxy.
+ 
+##The configuration format
+
+Before we continue, it is useful to take a step back and note that Velocity uses the [TOML](https://github.com/toml-lang/toml) format for its
+configuration. TOML was designed to be easy to understand, so you should not have difficulty understanding
+Velocity's configuration file.
+
+##Root section
+
+These settings mostly cover the basic, most essential settings of the proxy
+
+<style>
```

Comment:
These need to be global, per-theme styles

Reply:
fixed in 4ce76bcd44a19018a6228bc7df433f6eba042248